docs/user_guide/mitigation/reductions.rst

```diff
@@ -557,6 +557,39 @@ Group :code:`"a"` has an average loss of :math:`0.05`, while group
     detected by :code:`gamma` is identical to the mean absolute error.
 
 
+Exponentiated Gradient
+----------------------
+
+The ExponentiatedGradient algorithm in Fairlearn is used to produce models that 
+satisfy fairness constraints without needing access to sensitive features at deployment time.
+This algorithm creates a sequence of re-weighted datasets and retrains the 
+wrapped classifier on each of these datasets.
+To instantiate an ExponentiatedGradient model, you need to pass in a base estimator 
+and fairness constraints. The fairness constraints can be specified using an epsilon value,
+which represents the maximum allowed difference or ratio between the largest and smallest value.
+
+Here is an example of how to instantiate an ExponentiatedGradient model:
+
+.. doctest:: mitigation_reductions
```

Comment:
Instead of encapsulating inside `get_expgrad_models_per_epsilon`, let's just present this as a script. First, you could just call ` exp_grad_est.fit()`. Then report (or possibly plot) the (test or train) performance (in terms of accuracy and fairness) of the resulting ensemble, based on the predictions produced by `exp_grad_est.predict()`.

Then in the second part of the example, you can extract predictors `exp_grad_est.predictors_` and plot fairness vs accuracy of the individual models in the ensemble using [plot_model_comparison()](https://fairlearn.org/v0.10/api_reference/generated/fairlearn.metrics.plot_model_comparison.html)--perhaps also adding the ensemble predictions.